src/plugins/vis_type_table/public/index.scss

```diff
@@ -7,6 +7,6 @@
 //   tbvChart__legend--small
 //   tbvChart__legend-isLoading
 
-@import 'agg_table/index';
```

Comment:
The `styling_constants` import at the top of this file is no longer necessary.

Reply:
thx @cchaos, I've removed it